WMIAdventure/frontend/src/js/components/main-menu/atoms/Option/styled-components/OptionLink.js

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components';
+import {Link} from "react-router-dom";
+
+function alignSelfHandle(setSelfStart, setSelfEnd) {

```

Comment:
Czemu `handle`? To nie jest jakaś funkcja która wykonuje coś z zewnątrz

Reply:
setSelcStart i setSelfEnd otrzymuje z zewnątrz

Ale spoko to tylko nazwa, jak proponujesz inaczej nazwać?

Reply:
Po prostu `alignSelf`.

Czepiam się tylko, konwencja nazwenictwa funkcji z dopiskiem `handle` mówi o tym, że ta funkcja to "wystawienie dłoni" dla innej funkcji, która się wykona w ramach niej. Przykładem są propsy, gdzie props nazywasz `handle` żeby mu potem przekazać w tym propsie funkcje, która się wykona

Reply:
Ok, ja o tym nie wiedziałem. Nazywałem coś "handle" jak po prostu obsługiwało coś ze względu na coś innego z zewnątrz.